src/locales/es/translation.json

```diff
@@ -62,14 +62,16 @@
     "publisher": "Publicado por: {{publisher}}",
     "copy_link_to_clipboard": "Copiar enlace",
     "copied_link_to_clipboard": "Enlace copiado",
-    "hours": "horas",
-    "minutes": "minutos",
+    "hours": "hours",
+    "minutes": "minutes",
```

Comment:
Are these changes intended?

Reply:
it wasn't, some missclick i think, i rewrite them in spanish